lib/llvm-backend/src/lib.rs

```diff
@@ -31,23 +30,20 @@ pub type LLVMCompiler = SimpleStreamingCompilerGen<
     code::CodegenError,
 >;
 
-#[derive(Debug, StructOpt, Clone)]
+#[derive(Debug, Clone)]
 /// LLVM backend flags.
-pub struct CLIOptions {
+pub struct LLVMOptions {
```

Comment:
I'm confused why this struct exists.  Is it publicly exposed to users?  It seems identical to the one with structopt defined on it